### PR TITLE
Added IPv6 FDI options

### PR DESCRIPTION
**Pull request by lzap**
This adds two brand new options, do not merge yet the patch is still pending.

https://github.com/theforeman/foreman-discovery-image/pull/125

Adds few more options which are there. I am not adding all of them, most of them are pretty low-level and don't need to be customized:

https://theforeman.org/plugins/foreman_discovery/15.0/index.html#3.1.5Discoveryimagekerneloptions

### CODE REVIEW

File: guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc [hunk 1]
```diff
@@ -24,6 +24,21 @@ fdi.pxfactvalue1, fdi.pxfactvalue2 ... fdi.pxfactvalueN::
...
+  By default, the image initializes all network interfaces (value `all`). When this setting is set to `bootif`, only the network interface it was network-booted from will be initialized.
+
+fdi.rootpw::
+  By default, root account is locked. Use this option to set root password. Accepts both clear and encrypted passwords.
+
+fdi.ssh::
+  By default, the SSH service is disabled. Set this to `1` or `true` to enable SSH access.
+
+fdi.ipv4.method::
+  By default, NetworkManager IPv4 method setting is set to `auto`. This option overrides it, set it to `ignore` to disable IPv4 stack. This option only works in DHCP mode.
```
Comment (melcorr) [inline guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc:37]: Is there another word for "stack". Stack seems like one of those words that covers everything and nothing.

File: guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc [hunk 2]
```diff
@@ -24,6 +24,21 @@ fdi.pxfactvalue1, fdi.pxfactvalue2 ... fdi.pxfactvalueN::
...
+
+fdi.ipv6.method::
+  By default, NetworkManager IPv6 method setting is set to `auto`. This option overrides it, set it to `ignore` to disable IPv6 stack. This option only works in DHCP mode.
```
[the remark this answers is not in the document]
Comment (melcorr) [reply guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc:40]: If we just removed the word "stack" would anything change?

**Review by melcorr — CHANGES_REQUESTED** [review]
Looks great. Few suggested changes. One question: does any of this need markup for upstream? Will this be supported downstream?

**Comment by lzap** [comment]
All applied, please squash on commit thanks!

**Comment by melcorr** [comment]
Thank you!

**Pull request merged**